examples/file-and-cloudinary-upload/app/utils/utils.server.ts

```diff
@@ -0,0 +1,30 @@
+import dotenv from "dotenv";
+import cloudinary from "cloudinary";
+import type { Stream } from "stream";
+
+dotenv.config();
```

Comment:
Love this example! Just a question about environment variables: according to the docs `dotenv` should only be used when starting the development script via `node -r` (see [server environment variables](https://remix.run/docs/en/v1/guides/envvars#server-environment-variables)). However, I prefer your way much better, is there something wrong with it?

Reply:
I always avoid calling it via code. It's a potential source of bugs if you ship a ".env" to production by accident.

Reply:
Agreed! Fixed 👍